src/issue-creator/issue-reporter.ts

```diff
@@ -1,25 +1,42 @@
-import { HttpService } from "@nestjs/common"
+import { HttpService } from '@nestjs/common';
+import { request, gql } from 'graphql-request'
+import { IssueFormat } from '../IssueFormat';
+import { ConfigService } from '@nestjs/config';
 
-/*
-  Provides the basic funtionality every "detailed" IssueComponent should have
-*/
+/**
+ * Provides the basic functionality every "detailed" IssueComponent should have
```

Comment:
hier ist die rede von den IssueCreators, oder?
```suggestion
 * Provides the basic functionality every "detailed" IssueCreator should have
```